src/ivc/incrementally_verifiable_computation.rs

```diff
@@ -72,11 +72,11 @@ where
     C2::Base: PrimeFieldBits + FromUniformBytes<64>,
 {
     pub fn new<const T: usize, RP1, RP2>(
-        _pp: &mut PublicParams<C1, C2, RP1, RP2>,
+        _pp: &PublicParams<C1, C2, RP1, RP2>,
         _primary: SC1,
-        _z0_primary: [C1::Base; A1],
+        _z0_primary: [C1::Scalar; A1],
```

Comment:
Just want to confirm: `C1` is the curve for primary circuit, i.e. `C1::Scalar` is used for primary circuit constrain system?

Reply:
Yes